bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetAuxiliaryHandler.java

```diff
@@ -0,0 +1,137 @@
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+        OpenWebNetBridgeHandler h = bridgeHandler;
+        if (h != null && h.isBusGateway()) {
+            OpenGateway gw = h.gateway;
+            if (gw != null && gw.isConnected()) {
+                try {
+                    OpenMessage msg = BaseOpenMessage.parse("*#9##");
+                    send(msg);
+                } catch (MalformedFrameException | UnsupportedFrameException | OWNException e) {
+                    logger.warn("Exception while processing command {}: ", e.getMessage());
+                }
+            } else {
+                logger.warn("Gateway is not connected");
+            }
+        } else {
+            logger.warn("Gateway is not available");
+        }
+    }
+
+    /**
+     * Handles Auxiliary command for a channel
+     *
+     * @param channel the channel
+     * @param command the Command
+     */
+    @Override
+    protected void handleChannelCommand(ChannelUID channel, Command command) {
+        logger.debug("handleAuxiliaryCommand() (command={} - channel={})", command, channel);
+        Where w = deviceWhere;
+        if (w != null) {
+            if (channel.getId().equals(CHANNEL_AUX)) {
+                if (command instanceof StringType) {
+                    try {
+                        if (command.toString().equals(Auxiliary.WhatAuxiliary.ON.name())) {
+                            send(Auxiliary.requestTurnOn(w.value()));
+                        } else if (command.toString().equals(Auxiliary.WhatAuxiliary.OFF.name())) {
+                            send(Auxiliary.requestTurnOff(w.value()));
+                        }
+                    } catch (OWNException e) {
+                        logger.warn("Exception while processing command {}: {}", command, e.getMessage());
+                    }
+                } else {
+                    logger.warn("Unsupported command {} for channel {}", command, channel);
+                }
+            } else {
+                logger.warn("Unsupported ChannelUID {}", channel);
+            }
+        }
+    }
+
+    @Override
+    protected void requestChannelState(ChannelUID channel) {
+        /*
+         * NOTICE: It is not possible to get the state of a
+         * WHO=9 command. To get state of the Alarm system use WHO=5 instead
+         */
+    }
+
+    @Override
+    protected void refreshDevice(boolean refreshAll) {
+        /*
+         * NOTICE: It is not possible to refresh the state of a
+         * WHO=9 command. To refresh the state of the Alarm system use WHO=5 instead
+         */
+    }
+
+    @Override
+    protected Where buildBusWhere(String wStr) throws IllegalArgumentException {
+        return new WhereAuxiliary(wStr);
+    }
+
+    @Override
+    protected String ownIdPrefix() {
+        return Who.AUX.value().toString();
+    }
+
+    protected void handleMessage(BaseOpenMessage msg) {
+        super.handleMessage(msg);
```

Comment:
like this is useless. The method should be further implemented to handle AUX messages from the BUS. Is better to add a note to indicate support for managing AUX messages from BUS is still to be implemented.

Reply:
I would change comments as: 
* The {@link OpenWebNetAuxiliaryHandler} is responsible for sending Auxiliary (AUX) commands/messages to the bus
 * It extends the abstract {@link OpenWebNetThingHandler}.
 *
 * NOTICE: Support for handling messages from the BUS regarding alarm control has to be implemented.
 
 This in order to better clarify the feature

Reply:
ok. I also suggest to change the description of this PR

Reply:
I updated the description of this PR to limit support to sending AUX messages